src/components/Expenses/ExpenseDate.css

```diff
@@ -0,0 +1,26 @@
+.expense-date {
+  display: flex;
+  flex-direction: column;
+  width: 5.5rem;
+  height: 5.5rem;
+  border: 1px solid #ececec;
```

Comment:
Again I suspect that those colors will be reused many times in your app.
Please consider having global and local css variables for this